### PR TITLE
Analysis of static code

### PR DESCRIPTION
**Pull request by mafeZapata**

### CODE REVIEW

File: cypress/e2e/google.cy.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
...
-        cy.visit("https://www.google.com/");
-        cy.title().should("eq", "Google");
-    });
-});
+  it("should have a title", () => {
+    cy.visit("https://www.google.com/");
+    cy.title().should("eq", "Google");
+  });
+
+
```
Comment (AlejaGonzalezV) [inline cypress/e2e/google.cy.ts:7]: delete this empty line

**Review by AlejaGonzalezV — COMMENTED** [review]
Delete the screenshot file and add it in the PR description

**Comment by mafeZapata** [comment]
Evidence analiysis of static code

**Comment by mafeZapata** [comment]
This image is the evidence for the analysis of static code

File: tsconfig.json [hunk 2]
```diff
@@ -17,3 +17,5 @@
         "node_modules/"
     ]
 }
+
+
```
Comment (AlejaGonzalezV) [inline tsconfig.json]: Delete this empty line, just leave one

File: package.json [hunk 3]
```diff
@@ -24,7 +27,13 @@
   },
   "homepage": "https://github.com/mafeZapata/cypress-training#readme",
   "devDependencies": {
+
```
Comment (AlejaGonzalezV) [inline package.json]: delete this empty line

File: package.json [hunk 4]
```diff
@@ -4,8 +4,11 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test": "cypress open",
-    "test:open": "cypress open"
+    "test:open": "npm run lint && cypress open",
+    "lint": "eslint ./cypress/e2e/**/*.ts",
+    "lint:fix": "npm run lint -- --fix",
+    "test": "cypress open"
+
```
Comment (AlejaGonzalezV) [inline package.json]: delete this empty line

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**